data/virt_autotest/virt-tests.conf

```diff
@@ -0,0 +1,136 @@
+[vmi_lifecycle_test]
+GINKGO_FOCUS=vmi_lifecycle_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
```

Comment:
"-apply-default-e2e-configuration" is applied in all tests, will it be better to move it into the code block that generates the virt-tests command?

Reply:
Already moved this option into the code block.